addons/path-mapped/common/src/main/java/org/commonjava/indy/pathmapped/inject/PathMappedGroupRepositoryFilter.java

```diff
@@ -127,6 +144,16 @@ public boolean canProcess( String path, Group group )
                              .collect( Collectors.toList() );
     }
 
+    private boolean isMavenMetadataNonSnapshotPath( Group group, String path )
+    {
+        if ( group.getPackageType().equals( PKG_TYPE_MAVEN ) )
+        {
+            SpecialPathInfo pathInfo = specialPathManager.getSpecialPathInfo( path );
+            return pathInfo != null && pathInfo.isMetadata() && !path.contains( LOCAL_SNAPSHOT_VERSION_PART );
```

Comment:
I can not make sure if this is a right way to judge snapshot. Sometimes the snap path is something like A-V-xxxxxxx.xxxx(these x is some digits that represents temp date). Maybe ArtifactPathInfo.isSnapshot() is another way?

Reply:
I think the xxxx digits are for the artifacts pom/jar/etc. For metadata, it should be sth like, https://oss.sonatype.org/content/repositories/snapshots/org/commonjava/indy/launch/indy-launcher/1.9.5-SNAPSHOT/maven-metadata.xml

Reply:
This pr only works in case it is metadata.

Reply:
Ok. So it LGTM